### PR TITLE
Add config.md

### PR DESCRIPTION
**Pull request by DaveOkpare**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -31,97 +23,9 @@ You will also need to install [live_map](https://github.com/TGRHavoc/live_map) o
 
 Download the [latest version](https://github.com/TGRHavoc/live_map-interface/archive/master.zip).
 
-This should be enough to get the interface up and running on your website. If you want to change stuff like the images, look in the [configuration](#configuration) section
+This should be enough to get the interface up and running on your website. If you want to change stuff like the images, look in the [configuration](https://github.com/DaveOkpare/live_map-interface/blob/DaveOkpare-patch-1/docs/config.md) section
```
Comment (TGRHavoc) [inline docs/README.md:26]: This should be a relative link. `config.md` should be enough :)
```suggestion
This should be enough to get the interface up and running on your website. If you want to change stuff like the images, look in the [configuration](config.md) section.
```

**Review by TGRHavoc — CHANGES_REQUESTED** [review]

**Review by TGRHavoc — APPROVED** [review]
Looking good :)

**Comment by TGRHavoc** [comment]
Ah shoot. Forgot to ask. In future, would it be possible to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines?

So, documentation commit messages should look something like
```
docs: add docs/config.md
docs(readme): change spelling
docs(config): added table of contents
```
Etc. It just makes for easier reading of commits 🙂 

This is why I need to make a contributing file 😄

**Comment by DaveOkpare** [comment]
Alright. I'll follow the guidelines for future commits. 

Also, _thanks a lot for this opportunity, I'm a complete newbie but learning a lot from this, esp. with your feedbacks and I hope to learn more from you._

**Pull request merged**